### ISSUE TITLE
Hi, why does FPN module  inherit class "ScriptModule"  when gpu count is 1 ?

### ISSUE THREAD

**Issue by shoutOutYangJie** [dbolya/yolact#114]
> use_jit = torch.cuda.device_count() <= 1
> if not use_jit:
>     print('Multiple GPUs detected! Turning off JIT.')
> 
> ScriptModuleWrapper = torch.jit.ScriptModule if use_jit else nn.Module
> script_method_wrapper = torch.jit.script_method if use_jit else lambda fn, _rcn=None: fn

what is this paragraph?

**Comment by dbolya** [comment]
Read the answer in here: https://stackoverflow.com/questions/53900396/what-are-torch-scripts-in-pytorch

The reason I disable it when there are more than 1 GPU is because as of version 1.1.0, Pytorch still doesn't support DataParallel (the multi GPU stuff) in TorchScript.

**Comment by shoutOutYangJie** [comment]
> Read the answer in here: https://stackoverflow.com/questions/53900396/what-are-torch-scripts-in-pytorch
> 
> The reason I disable it when there are more than 1 GPU is because as of version 1.1.0, Pytorch still doesn't support DataParallel (the multi GPU stuff) in TorchScript.

thanks for your quick reply!  In addtion, I find some weird annatation ,like "    # What is this ugly lambda doing in the middle of all this clean prediction module code?"

notes that you are the  author of this code, why did you leave this annotation? You should be very clear about your project. I am just curious

**Comment by dbolya** [comment]
I think that's pretty clear: the lambda is ugly and it's in the middle of some clean code.

is joke haha 😂💦

**Comment by shoutOutYangJie** [comment]
> I think that's pretty clear: the lambda is ugly and it's in the middle of some clean code.
> 
> is joke haha 😂💦

ok, I notice that you didn't use semantic loss because use_semantic_loss is False in yolact_base_config. so the model weight you have released is trained without semantic loss?

**Comment by dbolya** [comment]
Not entirely sure what you're looking at. It's set to true right here:
https://github.com/dbolya/yolact/blob/29e809e563d0c3e6c47c5c6716b8cc81686a2b24/data/config.py#L625

Maybe you were looking at `coco_base_config`? That's just the defaults for all the settings. You want `yolact_base_config` below.
